addons/website_crm_partner_assign/wizard/crm_forward_to_partner.py

```diff
@@ -104,14 +104,11 @@ def action_forward(self):
         return True
```

Comment:
> Commit

Isn't the URL wrong, not just empty?

Reply:
No by default it is empty if you don't have the invisible field (so yes useful). Because without it, the computed url returned by the onchange is just ignored, and so the value is False -> Link Empty.

The case where you can see the wrong link is if you use the wizard in python (as we did in server action on our production)

Hard to test, since the problem is that if field is not in the view, the onchange don't keep the value. You cannot reproduce with just a create, you need to trigger onchange and co if you want to have the bug of empty url. And test that portal return a 200 (or create a complete tour) seems high cost for small check.